scripts/testnet.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# API used for funding the calls, ideally https://api.hoprnet.org
+declare API_ENDPOINT="https://api-hopr-oxbs3v49y-jjperezaguinaga.vercel.app"
```

Comment:
This should be replaced by `https://api.hoprnet.org` as soon as https://github.com/hoprnet/api/pull/7 is merged.